.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on: [push, pull_request]
+
```

Comment:
push? why?

Reply:
Just to make sure whenever commit/pr is pushed to any branch or master. Nobody breaks changes